stdlib/public/core/StaticBigInt.swift

```diff
@@ -0,0 +1,203 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+/// An immutable arbitrary-precision signed integer.
+///
+/// `StaticBigInt` is primarily intended to be used as the associated type of an
+/// `ExpressibleByIntegerLiteral` conformance.
+///
+///     extension UInt256: ExpressibleByIntegerLiteral {
+///         public init(integerLiteral value: StaticBigInt) {
+///             precondition(
+///                 value.signum() >= 0 && value.bitWidth <= 1 + Self.bitWidth,
+///                 "integer overflow: '\(value)' as '\(Self.self)'"
+///             )
+///             self.words = Words()
+///             for wordIndex in 0..<Words.count {
+///                 self.words[wordIndex] = value[wordIndex]
+///             }
+///         }
+///     }
+@available(SwiftStdlib 5.8, *)
+@frozen
+public struct StaticBigInt:
+  _ExpressibleByBuiltinIntegerLiteral,
+  ExpressibleByIntegerLiteral,
+  Sendable
+{
+  @available(SwiftStdlib 5.8, *)
+  @usableFromInline
+  internal let _value: Builtin.IntLiteral
+
+  @available(SwiftStdlib 5.8, *)
+  @inlinable
+  public init(_builtinIntegerLiteral value: Builtin.IntLiteral) {
+    _value = value
+  }
+
+  /// Returns the given value unchanged.
+  @_alwaysEmitIntoClient
+  @inline(__always)
+  public static prefix func + (_ rhs: Self) -> Self {
+    rhs
+  }
```

Comment:
Does this operator function need `@available(SwiftStdlib 5.8, *)`?

Reply:
> Does this operator function need `@available(SwiftStdlib 5.8, *)`?

@lorentey Please advise on this issue of `@_alwaysEmitIntoClient` availability, for an "identity function" that doesn't call any other APIs.

There was also a [comment](https://github.com/apple/swift/pull/40722#issuecomment-1237468945) below, concerning your coding style documentation.